tools/pot/README_dev.md

```diff
@@ -24,31 +24,30 @@ Post-Training Optimization Tool includes standalone command-line tool and Python
 - OpenVINO
 
 ### Installation (Temporary)
-1) Clone compression tool repo: `git clone git@gitlab-icv.inn.intel.com:algo/post-training-compression-tool.git`
+1) Clone DLDT repo: `git clone https://github.com/openvinotoolkit/openvino`
 2) Download submodules:
    ```
    git submodule init
    git submodule update
    ```
-3) Clone DLDT repo: `git clone https://github.com/openvinotoolkit/openvino` (Not into the post-training-compression-tool)
-4) Switch dldt to required branch: `feature/low_precision/develop_fp_v10`
-5) Build inference engine (Instruction can be found in dldt repo)
-6) Switch dldt to _mkaglins/poc_ branch (Inference engine is built from _feature/low_precision/develop_fp_v10_ branch to support `FakeQuantize` layers. ModelOptimizer is used from _mkaglins/poc_ branch. So stay on _mkaglins/poc_ branch as you've built IE and don't build it from there again)
-7) Set _PYTHONPATH_ variable: `export PYTHONPATH=<path to DLDT bins>/bin/intel64/Release/lib/python_api/python3.6:<path to DLDT>/dldt/model-optimizer`
-8) Install requirements for accuracy checker:
+3) Declare Model Optimizer. 
```

Comment:
Please change "Declare" to "Setup" everywhere.

Reply:
changed, thanks.